index.js

```diff
@@ -5,6 +5,8 @@ var File = require('vinyl');
 var path = require('path');
 var hljs = require('highlight.js');
 
+var pageIndex = [];
```

Comment:
This `pageIndex` variable is redundant to the existing `posts` variable [here](https://github.com/JosephDavis/gulp-flat-blog/blob/edaaa4caaa05269ef3f2ea7c29c467c3d4b61bae/index.js#L33).